worldengine/image_io.py

```diff
@@ -0,0 +1,215 @@
+"""
+This file is supposed to be the wrapper around image-processing modules like
+PyPNG or Pillow. These modules should not be included anywhere but here. Should
+a later replacement be necessary, it should be easy to do.
+This module provides elaborate means to write images and simple means to read
+them, too - the latter is only needed for the tests to be able to run, hence a
+rudimentary implementation should suffice.
+
+In case the used library was replaced, the following functions would have to be
+rewritten:
+  PNGWriter.complete()
+  PNGWriter.prepare_array()
+  PNGReader.__init__()
+"""
+
+import numpy
+import png
+#Documentation PyPNG: https://pythonhosted.org/pypng/png.html
+#Documentation PurePNG: http://purepng.readthedocs.org/en/latest/
+#The latter one is a fork of the former one. It is yet to be seen which one is better.
+
+class PNGWriter(object):

```

Comment:
Compared to the old ImageWriter-class this is restricted to PNG, but that one format it handles very well (due to PyPNG). I thought about adding in palette-support since most of worldengines output only uses a very small palette, but the class already seems like a bit of overkill and I didn't want to overcomplicate things.
If wanted, palette-support can go in, however.